bootstrap/main.css

```diff
@@ -0,0 +1,119 @@
+html,
+body {
+  margin: 0;
+  height: 100%;
+  overflow: hidden;
+  background: #0d6efd;
+  font-family: Inter, sans-serif;
+  position: relative;
+}
+
+.header {
+  color: #fff;
+  font-size: 2.4vh;
```

Comment:
For font-size, I would recommend using rem

Reply:
Adnan has not worked on the exact typography rules for this bootstrap theme. This is just a make-shift solution. So I think it's better to leave the unit as it is for now. Then change it when Adnan produces exactly what he has in mind.

Reply:
> For font-size, I would recommend using rem

Adnan has not worked on the exact typography rules for this bootstrap theme. This is just a make-shift solution. So I think it's better to leave the unit as it is for now. Then change it when Adnan produces exactly what he has in mind.